rewrite-java/src/main/java/org/openrewrite/java/JavaTemplate.java

```diff
@@ -29,12 +30,15 @@
 
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.HashSet;
 import java.util.Set;
 import java.util.function.Consumer;
 
-@SuppressWarnings("unused")
+@SuppressWarnings({"unused", "LombokGetterMayBeUsed"})
 public class JavaTemplate implements SourceTemplate<J, JavaCoordinates> {
+    @Getter
```

Comment:
Why?

Reply:
IJ was suggesting this change to me. It was flagging the method as a warning. I don't care either way, I was just trying to clean up the warning.

Reply:
If you want to just add a suppression on the `getCode` method, I'm also more than happy to do that instead